src/h3lib/lib/coordijk.c

```diff
@@ -207,6 +208,41 @@ void _ijkScale(CoordIJK *c, int factor) {
     c->k *= factor;
 }
 
+/**
+ * Returns true if _ijkNormalize with the given input could have a signed
+ * integer overflow. Assumes k is set to 0.
+ */
+bool _ijkNormalizeCouldOverflow(const CoordIJK *ijk) {
+    // Check for the possibility of overflow
+    int max, min;
+    if (ijk->i > ijk->j) {
+        max = ijk->i;
+        min = ijk->j;
+    } else {
+        max = ijk->j;
+        min = ijk->i;
+    }
+    if (min < 0) {
+        // Only if the min is less than 0 will the resulting number be larger
+        // than max. If min is positive, then max is also positive, and a
+        // positive signed integer minus another positive signed integer will
+        // not overflow.
+        if (SUM_INT32S_OVERFLOWS(max, min)) {
+            // max + min would overflow
+            return true;
+        }
+        if (SUB_INT32S_OVERFLOWS(0, min)) {
+            // 0 - INT32_MIN would overflow
```

Comment:
This comment should be:

```suggestion
            // 0 - min would overflow
```

Right?

Reply:
In this case, `min` would be `INT32_MIN`, which is I believe the only case that would hit this condition.